presto-docs/src/main/sphinx/release/release-0.287.rst

```diff
@@ -0,0 +1,102 @@
+* Add documentation for supported data `Type mapping <../connector/iceberg.html#type-mapping>`_  in the Iceberg connector. :pr:`22093`
+* Add histogram column statistic to Presto for the optimizer. Connectors can now implement support for them. :pr:`21236`
+* Add limit to the amount of data written during CTE Materialization. This is configurable by the session property ``query_max_written_intermediate_bytes`` (default is 2TB). :pr:`22017`
+* Add log of stats equivalent plan and canonicalized plan for HBO. This feature is controlled by session property ``log_query_plans_used_in_history_based_optimizer``. :pr:`22306`
+* Add optimization for query plans which contain RowNumber and TopNRowNumber nodes with empty input. :pr:`21914`
+* Add session property ``history_optimization_plan_canonicalize_strategy`` to specify the plan canonicalization strategies to use for HBO. :pr:`21832`
+* Add support for Iceberg connector in presto_protocol. :pr:`21765`
+* Add support for tracking of the input data size when there is a fragment result cache hit. This can be enabled by setting the configuration property ``fragment-result-cache.input-data-stats-enabled=true``. :pr:`22145`
+* Add usage documentation for :doc:`/clients/presto-cli`. :pr:`22265`
+* Add usage documentation for :doc:`/clients/presto-console`. :pr:`22349`
+* Add worker type and query ID information in HBO stats. :pr:`22234`
+* Remove ``native_execution_enabled``, ``native_execution_executable_path`` and ``native_execution_program_arguments`` session properties. Corresponding configuration properties are still available. :pr:`22183`
+* Remove the configuration property ``use-legacy-scheduler`` and the corresponding session property ``use_legacy_scheduler``.   The property previously defaulted to true, and the new scheduler, which was intended to replace it eventually, was never productionized and is no longer needed. The configuration property ``max-stage-retries`` and the session property ``max_stage_retries`` have also been removed. :pr:`21952`
+* CAST(str as INTEGER), CAST(str as BIGINT), CAST(str as SMALLINT), CAST(str as TINYINT) now allow leading and trailing spaces in the string. :pr:`22284`
+* Remove SPI method ConnectorMetadata.getTableLayouts() as deprecated. Add ConnectorMetadata.getTableLayoutForConstraint() as replacement. :pr:`21933`
+* Improve propagation of logical properties by enabling it by default. :pr:`22266`
+* Add support for Iceberg V1 tables in Prestissimo. :pr:`22013`
+* Add JSON as a supported output format in the Presto CLI. :pr:`22181`
+* Move `SortNode` to SPI module to be utilized in connector. :pr:`22497`
+* Add support for Apache DataSketches KLL sketch with the ``sketch_kll`` and related family of functions. :pr:`21568`
+* Optimize ``map_normalize`` builtin SQL UDF to avoid repeated reduce computation. :pr:`22211`
```

Comment:
Is ``map_normalize`` description as a "builtin SQL UDF" correct? If not, how should this be rephrased?